homeassistant/components/sonos/media_player.py

```diff
@@ -1245,4 +1275,13 @@ def device_state_attributes(self):
         if self._speech_enhance is not None:
             attributes[ATTR_SPEECH_ENHANCE] = self._speech_enhance
 
+        if self._queue is not None:
+            attributes[ATTR_QUEUE] = self._queue
+
+        if self._current_queue_position is not None:
+            attributes[ATTR_CURRENT_QUEUE_POSITION] = self._current_queue_position
+
+        if self._is_playing_local_queue is not None:
+            attributes[ATTR_IS_PLAYING_LOCAL_QUEUE] = self._is_playing_local_queue
```

Comment:
Also for keeping the state attributes smaller, I wonder whether this attribute could be removed and one could deduce the same information from the queue position being present or not.

Reply:
The queue is always present even when you are listening to radio
So I think this attribute is useful